test/ci-cd/scripts/comfyui/run_in_cluster.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# This script ALWAYS exits 0. All failures are logged to ERROR_FILE.
+
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+# --- Load env (best-effort) ---
+# shellcheck disable=SC1091
+source /workspace/build.env 2>/dev/null || true
+if [ -n "${ACP_PLATFORM_BASE_DIR:-}" ]; then
+  # shellcheck disable=SC1091
+  source "${ACP_PLATFORM_BASE_DIR}/use-cases/inference-ref-arch/terraform/_shared_config/scripts/set_environment_variables.sh" 2>/dev/null || true
+fi
+
+# --- Vars (preserved defaults) ---
+export POD_NAME="${POD_NAME:-comfyui-client}"
+export ERROR_FILE="${ERROR_FILE:-/workspace/build-failed.lock}"
+export TEST_WORKFLOW_DIR="${TEST_WORKFLOW_DIR:-test/ci-cd/scripts/comfyui/}"
```

Comment:
Removed the trailing `/` 
```suggestion
export TEST_WORKFLOW_DIR="${TEST_WORKFLOW_DIR:-test/ci-cd/scripts/comfyui}"
```